src/core/operation.c

```diff
@@ -119,6 +119,8 @@ QuicOperationFree(
             }
         } else if (ApiCtx->Type == QUIC_API_TYPE_STRM_RECV_SET_ENABLED) {
             QuicStreamRelease(ApiCtx->STRM_RECV_SET_ENABLED.Stream, QUIC_STREAM_REF_OPERATION);
+        } else if (ApiCtx->Type == QUIC_API_TYPE_STRM_PROVIDE_RECV_BUFFERS) {
+            QuicStreamRelease(ApiCtx->STRM_PROVIDE_RECV_BUFFERS.Stream, QUIC_STREAM_REF_OPERATION);
```

Comment:
What about `ApiCtx->STRM_PROVIDE_RECV_BUFFERS.Chunks`? We need to clean those up too, right?

Reply:
Operations can be canceled before they're ever processed. So, this needs to handle that case.